about.html

```diff
@@ -0,0 +1,77 @@
+        <li><a href="index.html">Program</a></li>
+        <li><a href="index.html">Join</a></li>
+        <li><a href="index.html">Sponsor</a></li>
+        <li><a href="index.html">News</a></li>
+        <li><a href="index.html">CC Campaign</a></li>
+      </ul>
+    </nav>
+    <article class="welcome-about">
+      <p class="hello hello-about">"Hello! Sharing World"</p>
+      <h1 class="main-heading">CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
+      <P class="intro-about">The Global Summit brings together the community of  
+        experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+        This year, we hope to expand our invitation list including organizations and individuals.</P>
+        <div class="please-contact-us">
+          <p class="contact-msg">Please contact us per Email for nay further questions about CC Global Summit 2015!</p>
+          <p class="contact-email">ccsummit2015@cckorea.org</p>
+        </div>
+       
+        <div class="logo-info">
+          <p class="logo-heading">Creative Commons 
+            Global Summit 2015 Logo</p>
+            <hr class="logo-heading-line">
+            <p class="logo-description">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.</p>
+            <img src="img/gs-logo.png" alt="" class="logo">
+        </div>
+       
+        <div class="previous-summit">
+            <div class="previous-summmit-texts">
+              <h3 class="previous-summit-heading">See the past CC Global Summits</h3>
+              <hr class="logo-heading-line">
+              <p class="previous-summit-intro">Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+            </div>
+            <div class="previous-summit-images">
+              <img src="img/about_2011.png" alt="2011 summit image" class="prev-summit-img">
+              <img src="img/about_2013.png" alt="2013 summit image" class="prev-summit-img">
+            </div>
+        </div>
+
+        <div class="partners">
+          <div class="partner-info">
+            <h3 class="partners-title  previous-summit-heading">Partner</h3>
+            <hr class="logo-heading-line">
+          </div>
+          <div class="logo-group">
+            <img src="img/mozilla-logo.png" alt="mozilla logo">
+            <img src="img/google-logo.png" alt="google logo">
+            <img src="img/naver.png" alt="naver logo">
+            <img src="img/orange_logo.png" alt="orange logo">
+            <img src="img/airbnb_logo.png" alt="airbnb logo">
+          </div>
+        </div>
+
+        <footer id="footer">
+          <div class="footer-content">
+            <img src="img/cck-last.png" alt="footer logo">
+            <p class="credit">2015 Creative Commons Korea. Some Rights Reserved.</p>
+          </div>
+        </footer>
```

Comment:
- [ ] On the about page, the footer has a bg-color dark not white, please refer to the original design and change it.

<img width="1190" alt="7" src="https://user-images.githubusercontent.com/77843050/211075964-b2882de5-5e08-455d-9493-3ad1dc5d2c1e.png">